samcli/cli/types.py

```diff
@@ -232,6 +239,17 @@ def convert(self, value, param, ctx):
 
         return result
 
+    def _add_value(self, result, key, new_value):
```

Comment:
Missing type hints.

Reply:
Yeah, the whole file is missing type hints, but I've added them for my additions